packages/editor-ui/src/components/Sticky.vue

```diff
@@ -232,14 +246,28 @@ export default defineComponent({
 			this.setForceActions(false);
 		},
 		async deleteNode() {
+			if (!this.data?.name) {
+				this.toast.showMessage({
```

Comment:
Can we remove this error and just return? this is a pretty unnecessary and unrealistic edge case.

Reply:
I was in between returning and notifying but am not too comfortable with silent returns. Another option that just popped into my mind, is using `assert` so it will throw an error if this ever happens. Will update